src/pages/interviews/softwareengineeringradio

```diff
@@ -0,0 +1,137 @@
+
+<iframe style="border-radius:12px" src="https://open.spotify.com/embed/episode/3pMpCqAoqaWI3vuWDz6lfp?utm_source=generator" width="100%" height="232" frameBorder="0" allowfullscreen="" allow="autoplay; clipboard-write; encrypted-media; fullscreen; picture-in-picture"></iframe>
+
+[Software Engineering Radio - Episode 515: Swizec Teller on Becoming a Senior Engineer](https://www.se-radio.net/2022/06/edpisode-515-swizec-teller-on-becoming-a-senior-engineer/)
+
+## Talking points
+
+* Swizec on being a software engineer, from basics to influencing leadership, mentorship, learning performance management and career path.
+* Career titles and what you should do with it.
+* Coding to solve problems
+* The main difference between a junior or mid-level engineer and a senior engineer.
+* Identifiable behaviors of a senior engineer.
+* Senior engineers work as a force multiplier
+* Codifying best practices
+* How to lead a project
+* Influencing other team members or teams to use your solution instead of writing their own
+* Timeboxing
+* Importance of mentorship
+* Performance Management - managing the performance of senior engineers
+* Growth and Career Progression for Senior Engineers
+* Skills to transition from junior to senior engineer
+* Working at a small company vs big company
+
+## Quotable Quotes
+
+"Years of tenure are probably the least correlated with you being a true senior software engineer" - ST
+
+"If you do get a good job title, I would definitely take it. Because titles do help your career. That's why we give titles out." - ST
+
+"Once you have that title, or if you want to get that title based on merit, rather than just years on the job, focus primarily on how you approach solving problems." - ST
+
+"Engineers like to get really focused on the engineering and coding aspects. We kind of forget about why it is that we do engineering and why it is that we write code" - ST
+
+"The more senior you get, the more you become focused on using code as a tool to solve problems and to achieve business results rather than code for its own sake." - ST
+
+"The main difference between a mid-level or a junior, is that they get more focused on the code and how to actually do the thing, rather than when you're more senior, when you become more focused on the problem solving and the business stuff" - ST
```

Comment:
```suggestion
"The main difference between a mid-level or a junior, is that they get more focused on the code and how to actually do the thing, rather than when you're more senior, when you become more focused on the problem solving and the business stuff" - ST
```